zkevm-circuits/src/evm_circuit/execution.rs

```diff
@@ -946,6 +955,83 @@ impl<F: Field> ExecutionConfig<F> {
         )
     }
 
+    fn annotate_circuit(&self, region: &mut Region<F>) {
+        let mut acc = 0usize;
+
+        self.advices.iter().enumerate().for_each(|(idx, &col)| {
+            if idx < FIXED_TABLE_LOOKUPS {
+                region.name_column(|| format!("EVM_lookup_fixed_{}", idx), col);
+                acc += FIXED_TABLE_LOOKUPS;
```

Comment:
Shouldn't this only be called once? (it also applies to the rest of the if cases):
```
if idx == FIXED_TABLE_LOOKUPS - 1 {
    acc += FIXED_TABLE_LOOKUPS;
}
```

Reply:
You can use this instead :dancing_men: 
```rust
        let groups = [
            ("EVM_lookup_fixed", FIXED_TABLE_LOOKUPS),
            ("EVM_lookup_tx", TX_TABLE_LOOKUPS),
            ("EVM_lookup_rw", RW_TABLE_LOOKUPS),
            ("EVM_lookup_bytecode", BYTECODE_TABLE_LOOKUPS),
            ("EVM_lookup_block", BLOCK_TABLE_LOOKUPS),
            ("EVM_lookup_copy", COPY_TABLE_LOOKUPS),
            ("EVM_lookup_keccak", KECCAK_TABLE_LOOKUPS),
            ("EVM_lookup_exp", EXP_TABLE_LOOKUPS),
            ("EVM_adv_phase2", N_PHASE2_COLUMNS),
            ("EVM_copy", N_COPY_COLUMNS),
            ("EVM_lookup_byte", N_BYTE_LOOKUPS),
            ("EVM_adv_phase1", 9999),
        ];
        let mut group_index = 0;
        let mut index = 0;
        for col in self.advices {
            let (name, length) = groups[group_index];
            region.name_column(|| format!("{}_{}", name, index), col);
            index += 1;
            if index >= length {
                index = 0;
                group_index += 1;
            }
        }
```

Reply:
Fixed in 1956a982